src/main/car/inventory/Vehicle.java

```diff
@@ -97,19 +111,36 @@ public void setAcquisitionDate(Date acquisitionDate) {
 		this.acquisitionDate = acquisitionDate;
 	}
 
+	public boolean getRentStatus() {
+		return rentStatus;
+	}
+
+	public void setRentStatus(boolean status) {
```

Comment:
3 methods are not needed for setting rented status. Please remove the setRentedStatus() method